.github/workflows/failure-notifier.yml

```diff
@@ -0,0 +1,67 @@
+# We've configured Renovate to open bump PRs for Remix dependencies within our
+# test fixtures. This workflow sends notifications when one of these PRs fails.
+
+name: Notify on Remix bump failures
+
+on:
+  check_suite:
+    types:
+      - completed
+jobs:
+  notify-on-failure:
+    if: >-
+      (github.event.check_suite.conclusion == 'failure' || github.event.check_suite.conclusion == 'timed_out')
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: Check out the repository
+        uses: actions/checkout@v2
+
+      - name: Check for required label
+        id: check_label
+        uses: actions/github-script@v6
+        with:
+          script: |
+            // See `renovate.json5` at project root.
+            const requiredLabel = "bump-framework-in-fixtures";
+            const pullRequest = github.event.check_suite.pull_requests[0];
+            if (!pullRequest) {
+              core.setOutput("label_exists", "false");
+              return;
+            }
+
+            const { data: labels } = await github.issues.listLabelsOnIssue({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: pullRequest.number,
+            });
+
+            const labelExists = labels.some(label => label.name === requiredLabel);
+            core.setOutput("label_exists", labelExists ? "true" : "false");
+
+      - name: Create issue on failure
+        if: ${{ steps.check_label.outputs.label_exists == 'true' }}
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const ISSUE_LABEL = "framework-bump-failure";
+            const issues = await github.issues.listForRepo({
+              owner: github.context.repo.owner,
+              repo: github.context.repo.repo,
+              state: "open",
+              labels: ISSUE_LABEL
+            });
+            if (issues.length > 0) {
+              console.log(`Open issue already exists: ${issue.data.html_url}`);
+              return;
+            }
+            const issue = await github.issues.create({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              title: "Possible regression with new framework release",
+              labels: ISSUE_LABEL,
+              body: `A framework release bump in test fixtures has failed. Check ${github.event.check_suite.pull_requests[0].url}.`
+            });
```

Comment:
Can we have multiple renovate PRs bumping remix open at the same time (for example one bumping to stable and one to unstable)? Or it only ever be 1 bump PR?

As is, this would only be able to report single PR url, but if there only can be one open at the same time, that would be fine. Otherwise maybe instead of direct link we could link to a PR list with filters for `bump-framework-in-fixtures` label and `status: failed` (or whatever was the filter for checks status of PR).

Either way not a blocker, just food for thought

Reply:
Good question. As far as I know, it would be one open remix bump PR at a time, except that if both a major bump and a minor/patch bump are available it would open the major bump PR and leave the minor/patch bump PR in a pending state, visible on the dependency dashboard, where a human could tick the checkbox to open that PR as well. In that case we would have two open remix PRs at at time.

This seems OK to me. The system doesn't need to be perfect. We just want a non-noisy, proactive notification system for regressions.